docs/applications/add-ssl-certificate.md

```diff
@@ -13,6 +13,8 @@ links:
     featured:
 ---
 
+Before issuing an SSL certificate, a valid DNS record pointing to the server is required. If you don't have one configured already, please follow the [Find Server IP Address](/docs/servers/find-server-ip-address) guide and configure a DNS record on your DNS service before continuing to follow this guide.
```

Comment:
Incomplete text for you to re-consider the tone and approach, please fine tune it before applying suggestion:

```suggestion
Before issuing an SSL certificate, a valid DNS record must be pointing to the server on which the application is deployed, so the SSL Certificated can be issued and validated for the configured application web domain. 
To configure a DNS record pointing to your servers:
1. [Find the IP Address of the server](/docs/servers/find-server-ip-address) where the application is running
3. Use a DNS service to configure a DNS record for your application domain
  1. If you don't have your own domains configured with a DNS service, you can use free services such as [FreeDNS](https://freedns.afraid.org/) or hire paid managed DNS services such as  AWS Route53 <make it a link ro route53>, Cloudflare <to do link here>, or any [notable managed DNS service provider](https://en.wikipedia.org/wiki/List_of_managed_DNS_providers) that suits your needs.
2. Follow the specific instructions on your chosen DNS service
3. Make your sure your DNS record is pointing to your Devopness managed server
  > This can be verified using a DNS propagation checker such as [whatsmydns](https://www.whatsmydns.net/)
...

Once you have your application DNS record correctly pointing to the IP address of your Devopness managed server, please follow the instructions below to issue a SSL Certificate to your application:
```

Reply:
Understood, I'll review the post and make changes.

Updated version coming soon

Reply:
Post updated, 2 notes:

1. Added a details section to enable hiding steps to configure a DNS record, but still leaving the open state as default
2. The styling of the summary was added artificially and would need an update on the CSS to look and feel like this

![image](https://github.com/devopness/devopness/assets/33532177/fd15e14b-03e9-419a-94ea-17cbf419288f)